core/trino-main/src/main/java/io/trino/sql/analyzer/StatementAnalyzer.java

```diff
@@ -882,23 +882,19 @@ protected Scope visitAnalyze(Analyze node, Optional<Scope> scope)
         @Override
         protected Scope visitCreateTableAsSelect(CreateTableAsSelect node, Optional<Scope> scope)
```

Comment:
Should we handle for `visitCreateTable` ?

Reply:
https://github.com/trinodb/trino/blob/0cb081ca969dee02f22afe6edd45e111e474f920/core/trino-main/src/main/java/io/trino/sql/analyzer/StatementAnalyzer.java#L1124-L1127

Reply:
It should be removed. this should be handled by the engine. I will verify if there exists any gap in the testing as I didn't encounter this issue.